MiniApp/Classes/MiniApp.swift

```diff
@@ -0,0 +1,25 @@
+/// Mini App Public API methods
+public class MiniApp: NSObject {
+
+    /// Fetch the List of [MiniAppInfo] information.
+    /// Error information will be returned if any problem while fetching from the backed
+    ///
+    /// - Parameters:
+    ///     -   completionBlock: A block to be called when list of [MiniAppInfo] information is fetched. Completion blocks receives the following parameters
+    ///         -   [MiniAppInfo]: List of [MiniAppInfo] information.
+    ///         -   Error: Error details if fetching is failed.
+    public class func listMiniApp(completionBlock: @escaping (Result<[MiniAppInfo], Error>) -> Void) {
```

Comment:
I think this can be just `list`. `listMiniApp` seems redundant because the class name is `MiniApp`.